crates/cdk-cli/src/sub_commands/dlc/nostr_events.rs

```diff
@@ -107,18 +107,76 @@ mod tests {
 
     #[tokio::test]
     async fn test_lookup_announcement_event() {
-        let announemtent_id =
+        let announcement_id =
             EventId::from_hex("d30e6c857a900ebefbf7dc3b678ead9215f4345476067e146ded973971286529")
                 .unwrap();
 
         let client = Client::new(&Keys::generate());
         let relay = "wss://relay.damus.io";
         client.add_relay(relay.to_string()).await.unwrap();
         client.connect().await;
-        let event = lookup_announcement_event(announemtent_id, &client)
+        let event = lookup_announcement_event(announcement_id, &client)
             .await
             .unwrap()
             .unwrap();
-        assert_eq!(event.id, announemtent_id);
+        assert_eq!(event.id, announcement_id);
+    }
+
+    #[test]
+    fn test_create_dlc_message_event() {
+        let keys = Keys::parse("4e111131d31ad92ed5d37ab87d5046efa730f192f9c8f9b59f6c61caad1f8933")
+            .unwrap();
+        let counterparty_pubkey =
+            PublicKey::parse("d71b2434429b0f038ed35e0e3827bca5e65b6d44d1af9344f73b20ff7ffa93dd")
+                .unwrap();
+        let msg = String::from("hello");
+
+        let msg = base64::encode(msg);
+        let encrypted_message =
+            nip04::encrypt(keys.secret_key().unwrap(), &counterparty_pubkey, &msg).unwrap();
+
+        let event = create_dlc_msg_event(&keys, msg.clone(), &counterparty_pubkey).unwrap();
+
+        assert_eq!(keys.public_key(), event.pubkey);
+        assert_eq!(Kind::Custom(8888), event.kind);
+        // I cant get content encryption to pass
+        //assert_eq!(encrypted_message, event.content);
```

Comment:
If you look at the source code for `nip04::encrypt` you'll see it generates a random number:
`encrypt_with_rng(&mut rand::thread_rng(), secret_key, public_key, content)`
Therefore, calling `encrypt` twice on the same message with the same keys will never result in the same result.